environment.yml

```diff
@@ -10,10 +10,12 @@ dependencies:
   - dask
   - pip:
     - pytest
+    - imagecodecs
     - intake # for reading scivision
     - torch==1.10.0 # install before cefas_scivision; it needs this version
     - scivision
     - scikit-image
     - setuptools==69.5.1 # because this bug https://github.com/pytorch/serve/issues/3176
+    - tiffile
```

Comment:
I think `tiffile` is deprecated and this should now be `tifffile` (3 fs)!

Reply:
but 

```
[~]$ pip install tiffile
Collecting tiffile
  Using cached tiffile-2018.10.18-py2.py3-none-any.whl.metadata (883 bytes)
Collecting tifffile (from tiffile)
  Downloading tifffile-2024.7.21-py3-none-any.whl.metadata (31 kB)
Collecting numpy (from tifffile->tiffile)
  Downloading numpy-2.0.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 60.9/60.9 kB 9.7 MB/s eta 0:00:00
Using cached tiffile-2018.10.18-py2.py3-none-any.whl (2.7 kB)
Downloading tifffile-2024.7.21-py3-none-any.whl (225 kB)
```

???

Reply:
fair enough!